sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -323,6 +323,40 @@ To see how to use the Document Translation client library with Azure Storage Blo
 for your containers, and download the finished translated documents, see this [sample][sample_translation_with_azure_blob].
 Note that you will need to install the [azure-storage-blob][azure_storage_blob] library to run this sample.
 
+## Advanced Topics
+
+The following section provides some insights for some of the advanced translation features such as glossaries and custom translation models.
+
+### **Glossaries**
+Glossaries are domain-specific dictionaries. For example, if you want to translate some medical-related documents, you may need support for the many words, terminology, and idioms in the medical field which you can't find in the standard translation dictionary or you simply need specific translation. This is why Document Translator provides support for glossaries. 
+
+#### **How To Create Glossary File**
+
+Document Translator supports glossaries in the following formats:
+
+|**File Type**|**Extension**|**Description**|
+|----------------|-------------|-------------|
+|Localization Interchange File Format|.xlf. , xliff|A parallel document format, export of Translation Memory systems. The languages used are defined inside the file.|
+|Tab Separated Values/TAB|.tsv/.tab|A tab-delimited raw-data file used by spreadsheet programs.
+
+For .tsv/.tab files, you can read more about the format  [here][tsv_files_wikipedia]
+, and you can also see a sample file [here][sample_tsv_file].
+
+#### **How Use Glossaries in Document Translator**
+In order to use glossaries with Document Translator, you first need to upload your glossaries file to a blob container, and then provide it's SaS url to Document Translator as in the code samples [sample_translation_with_glossaries.py][sample_translation_with_glossaries].
```

Comment:
I think we should make sure we be specific and say the SAS URL to the glossary file so people don't mistaken it for SAS URL to the container.